ui/src/app/store/staticroute/types/actions.ts

```diff
@@ -0,0 +1,17 @@
+import type { StaticRoute } from "./base";
+import type { StaticRouteMeta } from "./enum";
+
+export type CreateParams = {
+  destination: StaticRoute["destination"];
+  gateway_ip: StaticRoute["gateway_ip"];
+  metric: StaticRoute["metric"];
```

Comment:
It looks like this is an optional param: https://github.com/maas/maas/blob/master/src/maasserver/forms/staticroute.py#L40

Reply:
Good catch, I'd just used the model file and got it wrong